pkg/volume/cephfs/cephfs.go

```diff
@@ -262,7 +268,7 @@ func (cephfsVolume *cephfs) execMount(mountpoint string) error {
 	for i = 0; i < l-1; i++ {
 		src += hosts[i] + ","
 	}
-	src += hosts[i] + ":/"
+	src += hosts[i] + ":" + cephfsVolume.subdir
```

Comment:
Hm, then it will throw an error. :smile: I actually checked the validation for `VolumeMount.mountPath` and it is not validated except for its length. https://github.com/kubernetes/kubernetes/blob/master/pkg/api/validation/validation.go#L1050

Adding a validation is no problem. How far should we take this:

1. Add missing leading slash 
2. Add validation that checks for leading slash
3. Add validation that checks for a wellformed path